hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -611,9 +611,9 @@ private class Emit[C](
       case F64(x) =>
         present(pt, const(x))
       case s@Str(x) =>
-        present(pt, mb.addLiteral(x, coerce[PString](s.pType)))
+        present(mb.addLiteral(x, coerce[PString](s.pType)).get)
```

Comment:
`.get` shouldn't be necessary here.  There should be an implicit conversion from PValue to PCode.  If there's not, can we add it?

Reply:
Done